bedrock/firefox/templates/firefox/channel/android.html

```diff
@@ -52,7 +52,7 @@
     <div class="container">
       <ul>
         <li><a href="{{ firefox_url('android', 'notes', 'beta') }}">{{_('Release Notes')}}</a></li>
-        <li><a href="{{ firefox_url('android', 'all', 'beta') }}">{{_('All Languages and Builds')}}</a></li>
+        <li><a href="{{ firefox_url('android', 'all', 'beta') }}">{% if l10n_has_tag('firefox_channel_bug1387466') %}{{_('All Languages and Platforms')}}{% else %}{{_('All Languages and Builds')}}{% endif %}</a></li>
```

Comment:
Where will the `firefox_channel_bug1387466` l10n tag be defined? (I don't know how bedrock l10n works.)

Reply:
It will be manually added to .lang files by @peiying2 or @flodolo [like this](https://github.com/mozilla-l10n/www.mozilla.org/commit/69bcc5f49fbeb8273cb0a929bbaf5e8058fad15b). In English, tags are simply ignored and the primary (not fallback) string will always be used.